doc/presentation/start_presentation.txt

```diff
@@ -4,20 +4,22 @@ OSGeoLive
 
 .. revealjs:: OSGeoLive
     :title-heading: h2
-    :subtitle: An Open Source Geospatial GNU/Linux Distribution
+    :subtitle: Open Geospatial Distribution
```

Comment:
Distribution of what?
GNU/Linux should remain IMO

Reply:
I've changed to include "Linux" instead of GNU/Linux. I think our target audience of new users will just be confused by the branding debate between Linux and GNU/Linux. Also, saying GNU/Linux in a lightning talk adds words when we are trying to be very concise.